lib/puppet-strings/yard/handlers/ruby/type_base.rb

```diff
@@ -99,6 +107,8 @@ def set_values(node, object)
         end
       end
     end
+
+
```

Comment:
can delete this extra whitespace

Reply:
Hey @scotje, I think I was having some problem with my commits, but I think it is resolved now.  Extra lines are removed!